### DISCUSSION TITLE
Command shortcut

### DISCUSSION THREAD

**Discussion by AHoroszkiewicz** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#890] [Q&A]
I'm looking for a way to control my smart lights with pc. I don't like my pc "hearing" me all the time and typing a command every time i want to turn on lights is annoying. Is there a way to create a shortcut for a command or even add a command as a tile to windows start menu?

**Comment by sumeshir26** [comment]
No, no way as of now.
